scripts/prepare.sh

```diff
@@ -31,7 +31,7 @@ function download_iot_agent() {
   local iot_version="${1:-${AGENT_DEFAULT_VERSION}}"
   local iot_agent_download_url="${IOT_AGENT_DOWNLOAD_URL}${iot_version}${DOWNLOAD_URL_TAIL}"
 
-  curl -L ${iot_agent_download_url} -o $TMPDIR/datadog-agent.deb
+  curl -L ${iot_agent_download_url} -o $TMPDIR/datadog-iot-agent.deb
   pushd ${TMPDIR}
     dpkg -x datadog-agent.deb .
```

Comment:
```suggestion
    dpkg -x datadog-iot-agent.deb .
```

We'll have to change this her as well

Reply:
Correct!!